README.md

```diff
@@ -97,15 +123,52 @@ client = Client(url='https://conjur.myorg.com',
                 ca_bundle='/path/to/my/ca/bundle.pem')
 ```
 
-#### With `.netrc` and `.conjurrc` settings
+#### `.conjurrc` file
 
-Write the code same as in the first example but create the client with the following arguments:
+You can provide the connection details needed to connect to the Conjur endpoint (`url`, `account`, and `ca_bundle`) in 
+a file called `.conjurrc` instead of passing them in during initialization of the Client.
 
-```python3
+The `.conjurrc` file should be saved to your home directory and should contain `conjur_url`, 
+`conjur_account`, and`cert_file`.
+
+```
+# .conjurrc
+---
+cert_file: /Users/someuser/conjur-server.pem
+conjur_account: someaccount
+conjur_url: https://conjur-server
+```
+
+#### Storing credentials
+
+When using the SDK, you can keep credentials in the system's native credential store instead of passing them in 
+during initialization of the Client. By default, the Client will favor saving credentials (login ID and password) 
+to the credential store. If the detected credential store is not one we support or is not accessible, the 
+credentials will be written to a config file, netrc, in plaintext. 
+
+If written in the netrc it is highly recommended that you remove the file when done.
```

Comment:
```suggestion
If written in the netrc, it is strongly recommended that you log out when not using the Conjur CLI. This removes the credentials from the local cache.


```

Reply:
updated to "This removes the credentials from the `.netrc` file." because I think we should refrain from saying using "local cache"